tldw_Server_API/app/api/v1/endpoints/admin/admin_sessions_mfa.py

```
@@ -86,6 +89,30 @@ async def admin_get_user_mfa_status(
     )
 
 
+class BulkMfaStatusResponse(BaseModel):
+    mfa_status: dict[str, bool]
+
+
+@router.get("/users/mfa/bulk", response_model=BulkMfaStatusResponse)
+async def admin_get_bulk_mfa_status(
+    ids: str = Query(..., description="Comma-separated user IDs"),
+    principal: AuthPrincipal = Depends(get_auth_principal),
```

Ruff (0.15.7)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)